### ISSUE TITLE
[TEST] Run 8x8x8 to check the whole pipeline with non 1D files

### ISSUE THREAD

**Issue by garikoitz** [vistalab/PRFmodel#44]

**Comment by garikoitz** [comment]
@noahbenson , right now if the synthetic json file is not passed the system fails. I started checking popeye but I guess the problem is in all tools. The problem I see with this is that some of the params for solving are coming from the json file, so if we use a real file, we'll need to pass those variables in the prfanalyze-tool-config.json file. I am guessing that we can assume this, but checking with you before doing anything:

- If there is no json file, it should assume that it is external data, and continue
- There will be missing configs, so if there is no json file, we'll nee to check that all the minimally required values are being passed to the function or fail (some defaults are not valid)
- Can external data be any size? (I mean 1D or volume) I think  so, but let's discuss.
- The output files with the results will be the same as the input format, 1D or volume, but following the same naming convention

**Comment by noahbenson** [comment]
Hmm, I'm a bit confused:
* "synthetic json file" is the file output from prfSynth?
* when is that file not passed? (only when a default config file should be written out?)
* can you give me some examples of the variables we'd need to add to one of the prfanalyze-tool-config.json file?

**Comment by garikoitz** [comment]
On Mon, Jan 27, 2020 at 11:59 AM Noah C. Benson <notifications@github.com>
wrote:

> Hmm, I'm a bit confused:
>
>    - "synthetic json file" is the file output from prfSynth?
>
> >>>> Yes, the system should work even if this file it is not being passed,
so then the system should assume that it is real human participant BOLD
time series is being passed right?

>
>    - when is that file not passed? (only when a default config file
>    should be written out?)
>
> >>>> No, when prfsynth is not being used at all. We create manually a BIDS
directory with the Stimuli and the sub-subjectname/ses-sessionname and put
there our own dataset (HCP?), and if we run prfanalyze, it will check the
derivatives/prfsynth directory always looking for the json file, but it
does not exists



>
>    - can you give me some examples of the variables we'd need to add to
>    one of the prfanalyze-tool-config.json file?
>
> >>>> Most of them we can calculate (TR, time points, stim size in
pixels...), some they must be passed:
    Stimulus.ResizedHorz
    Stimulus.ResizedVert
    Stimulus.fieldofviewHorz
    Stimulus.fieldofviewVert
    Stimulus.spatialSampleHorz
    Stimulus.spatialSampleVert
    stimradius    = pm1.Stimulus.fieldofviewHorz/2;
Basically, if we know the fov (os stimDiameter) in degrees, could we
calculate the rest? I need to consider many other things to generate the
files, but for solving, not so much right?

Would you add a flag in prfanalyze-config-defaults like syntheticDataset =
true/false, so that we can ignore the prfsynth output?
